src/AppBundle/Entity/Translations/ClientTranslation.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace AppBundle\Entity\Translations;
+
+use Doctrine\ORM\Mapping as ORM;
+use Sonata\TranslationBundle\Model\Gedmo\AbstractPersonalTranslation;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="client_translation",
+ *     uniqueConstraints={@ORM\UniqueConstraint(name="lookup_unique_role_translation_idx", columns={
+ *         "locale", "object_id", "field"
+ *     })}
+ * )
+ */
+class ClientTranslation extends AbstractPersonalTranslation
```

Comment:
Sorry, I was wrong, he did not like Softdeleteable. And Softdeleteable in entity not added, so erroneously did translation.  Need to clear the table.